From de121f49278459ce30d65e511de222ebe67ca7bd Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 9 Dec 2013 11:39:27 +0800
Subject: [PATCH 05/22] ENGR00291254 Make default healthd not to log battery
 event

sabresd_battery will report power state change by power_supply_changed()
every 5 seconds

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 healthd/healthd_board_default.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/healthd/healthd_board_default.cpp b/healthd/healthd_board_default.cpp
index ed4ddb4..26714c3 100644
--- a/healthd/healthd_board_default.cpp
+++ b/healthd/healthd_board_default.cpp
@@ -25,5 +25,5 @@ void healthd_board_init(struct healthd_config*)
 int healthd_board_battery_update(struct android::BatteryProperties*)
 {
     // return 0 to log periodic polled battery status to kernel log
-    return 0;
+    return 1;
 }
-- 
1.8.0

